### ISSUE TITLE
Running UI fails with ipywidgets==7.0.0

### ISSUE THREAD

**Issue by robertnishihara** [ray-project/ray#903]
The following error occurred after upgrading to `ipywidgets==7.0.0`. Downgrading to 6.0.0 solves the problem. We should be compatible with the latest version (and also with 6.0.0).

```
---------------------------------------------------------------------------
TraitError                                Traceback (most recent call last)
<ipython-input-4-bd7fdecbffea> in <module>()
----> 1 ui.task_timeline()

~/Workspace/ray/python/ray/experimental/ui.py in task_timeline()
    376     label_options = widgets.Label(value='View options:',
    377                                   layout=widgets.Layout(width='100px'))
--> 378     start_box, end_box, range_slider, time_opt = get_sliders(False)
    379     display(widgets.HBox([label_tasks, task_dep, label_objects, obj_dep]))
    380     display(widgets.HBox([label_options, breakdown_opt]))

~/Workspace/ray/python/ray/experimental/ui.py in get_sliders(update)
     58         orientation="horizontal",
     59         readout=True,
---> 60         readout_format=".0i%",
     61     )
     62 

~/anaconda3/lib/python3.6/site-packages/ipywidgets/widgets/widget_int.py in __init__(self, *args, **kwargs)
    245             kwargs['value'] = (0.75 * min + 0.25 * max,
    246                                0.25 * min + 0.75 * max)
--> 247         super(_BoundedIntRange, self).__init__(*args, **kwargs)
    248 
    249     @validate('min', 'max')

~/anaconda3/lib/python3.6/site-packages/ipywidgets/widgets/widget_int.py in __init__(self, value, **kwargs)
     73         if value is not None:
     74             kwargs['value'] = value
---> 75         super(_Int, self).__init__(**kwargs)
     76 
     77 

~/anaconda3/lib/python3.6/site-packages/ipywidgets/widgets/widget.py in __init__(self, **kwargs)
    409         """Public constructor"""
    410         self._model_id = kwargs.pop('model_id', None)
--> 411         super(Widget, self).__init__(**kwargs)
    412 
    413         Widget._call_widget_constructed(self)

~/anaconda3/lib/python3.6/site-packages/traitlets/traitlets.py in __init__(self, *args, **kwargs)
    995             for key, value in kwargs.items():
    996                 if self.has_trait(key):
--> 997                     setattr(self, key, value)
    998                 else:
    999                     # passthrough args that don't set traits to super

~/anaconda3/lib/python3.6/site-packages/traitlets/traitlets.py in __set__(self, obj, value)
    583             raise TraitError('The "%s" trait is read-only.' % self.name)
    584         else:
--> 585             self.set(obj, value)
    586 
    587     def _validate(self, obj, value):

~/anaconda3/lib/python3.6/site-packages/traitlets/traitlets.py in set(self, obj, value)
    557 
    558     def set(self, obj, value):
--> 559         new_value = self._validate(obj, value)
    560         try:
    561             old_value = obj._trait_values[self.name]

~/anaconda3/lib/python3.6/site-packages/traitlets/traitlets.py in _validate(self, obj, value)
    589             return value
    590         if hasattr(self, 'validate'):
--> 591             value = self.validate(obj, value)
    592         if obj._cross_validation_lock is False:
    593             value = self._cross_validate(obj, value)

~/anaconda3/lib/python3.6/site-packages/ipywidgets/widgets/trait_types.py in validate(self, obj, value)
    170         re_match = _number_format_re.match(value)
    171         if re_match is None:
--> 172             self.error(obj, value)
    173         else:
    174             format_type = re_match.group(9)

~/anaconda3/lib/python3.6/site-packages/traitlets/traitlets.py in error(self, obj, value)
    623             e = "The '%s' trait must be %s, but a value of %r was specified." \
    624                 % (self.name, self.info(), repr_type(value))
--> 625         raise TraitError(e)
    626 
    627     def get_metadata(self, key, default=None):

TraitError: The 'readout_format' trait of an IntRangeSlider instance must be a valid number format, but a value of '.0i%' <class 'str'> was specified.
```

cc @alanamarzoev @Wapaul1

**Comment by robertnishihara** [comment]
Even with #925, there are still some problems with the UI, for example the check boxes do not appear in the task timeline visualization.

**Comment by robertnishihara** [comment]
The error in the original post is fixed by #925.
